concordium-consensus/src/Concordium/Kontrol/Bakers.hs

```diff
@@ -311,7 +312,8 @@ getSlotBakersP4 ::
     forall m.
     ( BlockStateQuery m,
       PVSupportsDelegation (MPV m),
-      ChainParametersVersionFor (MPV m) ~ 'ChainParametersV1
+      ChainParametersVersionFor (MPV m) ~ 'ChainParametersV1,
+      SeedStateVersionFor (MPV m) ~ 'SeedStateVersion0
```

Comment:
Should we have a `SupportsSeedStateV0`? And likewise `SupportsSeedStateV1` ?

Reply:
I don't think it would be particularly useful to have it as an alias for this. We have `IsConsensusV0`, which includes the seed state constraint and is used at the higher levels.